product_docs/docs/biganimal/release/pricing_and_billing/index.mdx

```diff
@@ -70,6 +70,10 @@ To get a better sense of your AWS costs, check out the AWS pricing [calculator](
 
 Enabling [Apache Superset](/biganimal/latest/using_cluster/06_analyze_with_superset/) to analyze your data has an additional cost. In most cases the costs are approximately $150 per month, based on your cloud provider, instance and storage type selections, and other factors.
 
+## PgBouncer costs
+
+Enabling [PgBouncer](../getting_started/creating_a_cluster/#pgbouncer) to pool your connections incurs additional costs that depend on your cloud provider. Also, PgBouncer connects to your primary and requires an IP address; This IP costs $0.004 per hour. BigAnimal provisions up to three instances per PgBouncer-enabled cluster to ensure that performance is unaffected, so each availability zone receives its own instance of PgBouncer. 
```

Comment:
```suggestion
Enabling [PgBouncer](../getting_started/creating_a_cluster/#pgbouncer) to pool your connections incurs additional costs that depend on your cloud provider. In addition to the cloud provider costs, PgBouncer connects to your primary server and requires an IP address. This IP address costs $0.004 per hour. BigAnimal provisions up to three instances per PgBouncer-enabled cluster to ensure that performance is unaffected, so each availability zone receives its own instance of PgBouncer. 
```

Reply:
I'm good with most of this section but am requesting that we remove " This IP costs $0.004 per hour."  Resource charges fluctuates between CSPs, and can further fluctuate depending on the region where the resource is located.  Therefore, I think we can remove that one sentence.

Reply:
Thanks Jason